bin/node/src/aleph_cli.rs

```diff
@@ -47,9 +47,13 @@ pub struct AlephCli {
     #[clap(long, default_value_t = false)]
     enable_pruning: bool,
 
-    /// Maximum bit-rate per node in bytes per second of the alephbft validator network.
-    #[clap(long, default_value_t = 64 * 1024)]
-    alephbft_bit_rate_per_connection: u64,
+    /// Maximum bit-rate in bits per second of the alephbft validator network.
+    #[clap(long, default_value_t = 768 * 1024)]
+    alephbft_network_bit_rate: u64,
+
+    /// Maximum bit-rate in bits per second of the substrate network.
+    #[clap(long, default_value_t = 5*1024*1024)]
```

Comment:
How do we come to reason those defaults are sensible?

Reply:
For normal operation it should be plenty. While in major-sync mode, both local tests and on `t3a.xlarge` machines were unable to use more. Using a greater value I was able to kill a node on a `t3a.xlarge` machine by flooding. I can put some additional comment there.

Reply:
Okay